src/content/uk/2020/page-weight.md

```diff
@@ -0,0 +1,156 @@
+---
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+title: Розмір Сторінки
+description: Розділ "Розмір Сторінки" у 2020 Web Almanac розкриває, чому розмір сторінки важливий, пропускну здатність, складні сторінки, розмір сторінки з часом, запити сторінки та формати файлів.
+authors: [henrihelvetica]
+reviewers: [paulcalvano]
+analysts: [paulcalvano]
+editors: [tunetheweb]
+translators: [tymosh]
+henrihelvetica_bio: Henri - розробник-фрілансер, який перетворив свої захоплення на попурі з інженерії продуктивності, приправленого дрібкою користувацького доступу. У вільний від читання щоденного потоку дослідницьких документів і тематичних досліджень або не систематизованого аудиту сайтів у Dev Tools час, Henri робить внесок у спільноту, приймає участь у зустрічах з групового програмування, включаючи <a href="https://twitter.com/towebperf">Toronto Web Performance Group</a> або добровільно віддає свій обідній час і навчається у всіляких таборах. В інший час він бавиться з програмним забезпечення для створення музики, або найімовірніше тренується і фокусується на тому, щоб здолати найшвидше 5-кілометрівку.
+discuss: 2054
+results: https://docs.google.com/spreadsheets/d/1wG4u0LV5PT9aN-XB1hixSFtI8KIDARTOCX0sp7ZT3h0/
+featured_quote: Шлях Інтернету від простої, майже педагогічної платформи до інноваційних, складних і надзвичайно інтерактивних застосунків, які стали нормою, рудиментарний показник розміру сторінки приховав за собою більшу історію: рататуй з ресурсів, кожен з яких впливає на сучасні показники, в свою чергу впливаючи на користувацький досвід.
```

Comment:
This is your problem. You can't use colons in YAML as they mean something special so have to use the HTML entity instead:

```suggestion
featured_quote: Шлях Інтернету від простої, майже педагогічної платформи до інноваційних, складних і надзвичайно інтерактивних застосунків, які стали нормою, рудиментарний показник розміру сторінки приховав за собою більшу історію:&colon; рататуй з ресурсів, кожен з яких впливає на сучасні показники, в свою чергу впливаючи на користувацький досвід.
```

After that it seems to generate fine.